src/actions/task-definitions.js

```diff
@@ -198,6 +198,22 @@ tasks.changeToPluginCell = new UserTask({
   callback() { dispatcher.changeCellType('plugin') },
 })
 
+tasks.includeCellInRunAll = new UserTask({
```

Comment:
I do like that these are explicitly two tasks, but another possibility is to simply have this toggleable, which would make possible the keybinding `L` or something like that (maybe `S` for "skip" is a good key, but I'm worried about folks who were taught to keep their left hands on ASDF (like me) would accidentally hit it, but whatever)

Reply:
sure, i can make this task a toggle

"S" is fine with me -- i can add that if you wish. I'm not sure about the balance between the value of adding shortcuts and the cost of making the kbd a minefield of accidental actions (as you say). we may not be ther yet, so perhaps "S" is fine? i don't have a strong intuition on this

Reply:
actually, i want to have the two separate tasks for use in the menus, but I can add another "toggle skip" task to handle the keybinding.

Reply:
rechanged my mind, one toggle task is best :-)

Reply:
Yep, totally fine skipping the keyboard shortcut for now, unless it ends up being very obviously needed.

Reply:
oh i actually went ahead and added it :-) we can remove if we don't like it